src/components/contact.js

```diff
@@ -0,0 +1,84 @@
+import React from "react"
+import {
+  Heading,
+  Text,
+  Container,
+  Box,
+  FormErrorMessage,
+  FormLabel,
+  FormControl,
+  Input,
+  Button,
+  Center,
+} from "@chakra-ui/core"
+import emailjs from "emailjs-com"
+
+export default class ContactForm extends React.Component {
+  render() {
+    function sendEmail(e) {
+      e.preventDefault()
+
+      emailjs
+        .sendForm(
+          "service_7fvgwqf",
+          "template_vwtrhbo",
+          e.target,
+          "user_5ER6plW1UcDspryz51fCw"
+        )
+        .then(
+          result => {
+            console.log(result.text)
+          },
+          error => {
+            console.log(error.text)
+          }
+        )
+    }
+
+    return (
+      <div>
+        <Heading as="h2" size="lg" textAlign={["left", "center"]}>
+          Contact Us
+        </Heading>
+        <br />
+
+        <center>
+          <Box padding="8" bg="cyan.100" width="92vw" centerContent>
+            <form className="contact-form" onSubmit={sendEmail}>
+              <FormControl>
+                <FormLabel htmlFor="name">Full Name</FormLabel>
+                <Input name="name" placeholder="Enter your full name" />
+              </FormControl>
+              <br />
+              <FormControl>
+                <FormLabel htmlFor="email">Email</FormLabel>
+                <Input name="email" placeholder="Enter your email" />
+              </FormControl>
+              <br />
+              <FormControl>
+                <FormLabel htmlFor="message">Message</FormLabel>
+                <Input
+                  name="message"
+                  height="250px"
+                  placeholder="Enter message"
+                />
+              </FormControl>
```

Comment:
Form not labeled correctly. `htmlFor` should refer to the `id` of the input, but none of the input elements were given an `id` value.
For example, the Chakra example uses an `htmlFor` of `email` for the label, and sets the `id` of the input to `email`
![image](https://user-images.githubusercontent.com/11780062/95518201-971af900-0990-11eb-94b4-5294c56f9550.png)

![image](https://user-images.githubusercontent.com/11780062/95518131-68048780-0990-11eb-8cba-abdb2aea1b63.png)
See: https://chakra-ui.com/formcontrol#usage, https://storage.googleapis.com/lighthouse-infrastructure.appspot.com/reports/1602193254435-48910.report.html#accessibility